variables.tf

```diff
@@ -6,7 +6,7 @@ variable "bucket_name" {
 variable "additional_tags" {
   type        = map(string)
   description = "Additional tags to be attached to the S3 bucket."
-  default     = {}
+  default     = { Author : "Tamr" }
```

Comment:
I'm not sure I agree with setting this as the default, but I'm open to it.

If we do set this as the default, we should maybe call this a major version since anyone that's not currently using this field may accidentally get this new tag. Not catastrophic, but potentially unexpected.

Users that are already setting tags are going to completely miss this anyway, so benefit may be lost.

Maybe the better place to include this would be the examples to establish this as a best practice, both in this module and the tamr config module.
https://github.com/Datatamer/terraform-aws-s3/tree/master/examples
https://github.com/Datatamer/terraform-aws-tamr-config/tree/master/examples

If we do, maybe to be safe we would consider this a major version in hopes the person upgrading pays attention to the changelog.

Reply:
All good points, @souza-dan. We were thinking that modules could introduce tags, which we know to be a good practice, without requiring users to do anything special on their side. That way, everyone would get tagged resources out of the box and be able to reap the benefits (i.e. cost attribution, tag-based authorization, etc.).

But on second thoughts, yes, we should probably keep the default value an empty map and update the examples to establish this as a best practice.

Reply:
Great! Examples seem like a great place to introduce it then instead of defaults.